commentManager.js

```diff
@@ -7,11 +7,9 @@ var shared = require('./static/js/shared');
 
 exports.getComments = async function (padId, callback)
 {
-  // We need to change readOnly PadIds to Normal PadIds
-  var isReadOnly = padId.indexOf("r.") === 0;
-  if(isReadOnly){
-    padId = await readOnlyManager.getPadId(padId);
-  };
+  // We might need to change readOnly PadIds to Normal PadIds
+  var padIds = await readOnlyManager.getIds(padId);
```

Comment:
I changed each occurence of readOnlyManager to use the `getIds` method. That way both ids are available independent of the given input id.